Client/DataAccess/SQL/HistoryTable.swift

```diff
@@ -5,111 +5,59 @@
 let TableNameHistory = "history"
 let NotASiteErrorCode = 100
 
-class HistoryTable: Table {
-    let name = "History"
-    private let rows = "guid TEXT NOT NULL UNIQUE, " +
+class HistoryTable<T>: GenericTable<Site> {
+    override var name: String { return TableNameHistory }
+    override var rows: String { return "guid TEXT NOT NULL UNIQUE, " +
                        "url TEXT NOT NULL UNIQUE, " +
-                       "title TEXT NOT NULL"
-
-    let debug_enabled = false
-    func debug(msg: String) {
-        if debug_enabled {
-            println("HistoryTable: \(msg)")
+                       "title TEXT NOT NULL" }
```

Comment:
We probably also want a numeric primary key, so that visits can refer to it without using a GUID or an implicit _rowid.